joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/passes/controlflow/cfgdominator/CfgDominator.scala

```diff
@@ -28,9 +28,9 @@ class CfgDominator[NodeType](adapter: CfgAdapter[NodeType]) {
     val dominators = Array.fill(indexOf.size)(UNDEFINED)
     dominators(indexOf(cfgEntry)) = indexOf(cfgEntry)
 
-    /** Retrieve index of immediate dominator for node with given index. If the index is `UNDEFINED`, UNDEFINED is
-      * returned.
-      */
+    /* Retrieve index of immediate dominator for node with given index. If the index is `UNDEFINED`, UNDEFINED is
```

Comment:
what kind of warning did this trigger? maybe just some incorrect indentation?
context: we should really use `/** ... */` so that this is included in scaladoc

https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html

Reply:
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/16919236/199266976-e8869a2d-7f4f-4c88-8a3c-85a03d5258ad.png">
weird

Reply:
Strange, I don't get that warning...

I guess life is like a box of chocolates, sometimes ¯\_(ツ)_/¯

Reply:
oh, actually, I do get that warning - ignore my comment above - sorry! Will take a look - not that this is critical, but I'm curious

Reply:
actually, your first change was correct - the warning comes up because this looks like scaladoc, but since it's on a method inside a method, it'll never end up in scaladoc

I didn't see that because I didn't look at the surrounding context. 

I'll reapply your initial change

Reply:
haha, learned

Reply:
https://github.com/joernio/joern/pull/1929